alembic/versions/b99155654de_sunset_all_future_metrofirefox.py

```diff
@@ -0,0 +1,27 @@
+"""sunset all future MetroFirefox
+
+Revision ID: b99155654de
+Revises: 56f5cdf9bcdb
+Create Date: 2015-07-31 15:51:26.282796
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = 'b99155654de'
+down_revision = '56f5cdf9bcdb'
+
+from alembic import op
+
+
+def upgrade():
+    op.execute("""
```

Comment:
you also need `load_stored_proc(op, ['update_product_versions.sql'])`, to load the stored proc